### PR TITLE
Getmoreans Round Two. You can kill a man, but not an idea.

### PR DESCRIPTION
**Pull request by TehFlaminTaco**
Adds the living cans, "Getmoreans" as a playable race.

## Traits
Positive Traits:
+Immune to Oxygen, Toxin, and Radiation effects
+Has no eyes, so is immune to flashing / welding
+Resistant to Temperature damage
+No internal organs to look after. Just a Sweet-tasting Fizzy sludge.
+Feels no pain
+Doesn't need to eat/drink
Negative Traits:
-VERY weak to brute-force
-Cannot be cloned
-Low pay
-No head, Cannot wear headsets / helmets / masks
-Brand is always recognizable. Cannot hide their face.
-Cannot consume pills/liquids to gain their benefits. (Still weak to injections)

## Screenshots
Character Setup

Blank Can

Janican

## Description
```
The Getmore Chocolate Corperation, A Subsidiary of Nanotrasen, attempted to produce a
genetically modified line of sodas in order to dynamically appeal to customers. The Result? Getmoreans.
A sentient line of soda products with human features, these human-can hybrides are famous for speaking
corperate double speak, and generally being creepy. Instead of organs, they have a sweet tasting yet fizzy
syrup that seems to serve the same functions. Because of this, things like temperature and radation have little
effect on the cans, but being made of thin aluminum makes them extra weak to punches.
```

Getmoreans are the race of Can-people we've all been waiting for. Human-can hybrids that are proud of their brand, whatever absurd thing that may be. They lack a head, or anything resembling organs, and instead rely on their "Life Juice", the sweet-tasting fizzy fluid inside them which seems to serve multiple purposes within the Getmorean.

The name Getmorean stems from Getmore, the company who created them alongside Robust Industries in an attempt to make a new dynamic line of softdrinks. As such, Getmoreans are also typically trained in "Corporate Doublespeak", a twist on Basic which seems to be saying nothing in particular, but business professionals swear by.

The Average Getmorean names them-selves like a flavour of softdrink (Typically things like "Lemon Swirl" or more esoteric things), but the rare Getmorean will try to have a human name, to match the rest of their features. Getmoreans are also soully masculine, and cannot naturally reproduce. All Getmoreans are produced at Getmore's Getmorean Production Facilities™.

Getmoreans are usually loud, self-confident, and arrogant, with strong brand-loyalty to their own flavour, and Getmore. As they are technically products of Getmore, their contract reflects this, and they receive reduced pay.

## Q&A
- How do I play as one of these fantastic cans?
   - Apply on the forums.
- Do we need another admin-only joke race?
   - This is a whitelisted playable race.
- Does this fit our established lore or canon?
    - Established lore or canon is ours to decide.
- Please stop trying to make cans happen
    - That's not a question.

## Improvements from [Attempt One](https://github.com/Aurorastation/Aurora.3/pull/5752)
- Up to date with current code
- Better implemented following conventions better.
- Cleaner PR, only containing actually needed Commits this time.

Please address any concerns with Getmoreans realistically and professionally.

### CODE REVIEW

**Comment by TehFlaminTaco** [comment]
!review

File: code/modules/mob/living/carbon/human/species/station/getmorean/getmorean.dm [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+	max_nutrition_factor = -1
+
+	flesh_color = "#333333"
+
+	mob_size = MOB_SMALL
+
+	hud_type = /datum/hud_data/getmorean // Doesn't have a head. No head slots.
+
+	brute_mod =     3.0                  // Physical damage multiplier.
+	burn_mod =      0.3                  // Burn damage multiplier.
```
Comment (Arrow768) [inline code/modules/mob/living/carbon/human/species/station/getmorean/getmorean.dm:26]: This seems excessive and will most likely need to be tweaked a bit

Comment (TehFlaminTaco) [reply code/modules/mob/living/carbon/human/species/station/getmorean/getmorean.dm:26]: Solo balance is hard, but I feel it's plenty considering their other resistances.

**Comment by alsoandanswer** [comment]
Should probably adjust the icon since we have new soda sprites

**Comment by Kaedwuff** [comment]
Setting aside the personally baffling concept that this PR has not already been rejected by a dev out of hand for being blatantly, unsustainably silly, there are a number of design and technical issues with this race.  I've had some people suggest that this is a gag PR, and while I'd like to believe that, you seem to be taking yourself entirely seriously, and Arrow is in here giving balance reviews as if this isn't a joke.  

So, I guess we're supposed to not treat this like a joke, or perhaps I'm a humorless killjoy robot who hates fun. But I'd rather be called that than risk an chance that something like this legitimately gets added to the server.

Aesthetically, all you have done is replace the head and most of the torso of the base human sprite with a giant can, which is gaudy and not pleasant to look at.  It is also not particularly a fresh or novel as a concept.  You haven't introduced anything with 'Getmoreans' that creates a new culture or somehow enhances the server beyond the brief OOC amusement at seeing a walking can on the station.  They don't sound like they will contribute more of value other than to be a walking advertisement for their company, which is likely to get irritating extremely fast.

You've also misspelled "Corporate" on a number of occasions both in your sample images and in the content of your PR.  While we're on the subject, spouting a list of corporate buzzwords is not even a real form of communication in _this_ world, and the idea that we're supposed to listen to can people say strings of words like "Progress robust chocolate synergy promotion!" and not have our suspension of disbelief shattered into a million pieces just doesn't compute for me.

I can't escape the feeling and hope that perhaps you posted this on the wrong github by accident, and maybe you meant for this go to somewhere with a lower roleplay standard, like Citadel, or Goon.

**Comment by NonQueueingMatt** [comment]
No thank you.

**Pull request closed without merging**